### ISSUE TITLE
NullPointerException

### ISSUE THREAD

**Issue by Technici4n** [comofastm/ArsTheurgia#4]
Hello, this mod is causing a NPE. I don't have the full log, but this should be enough:
```
java.lang.NullPointerException: Ticking entity
    at net.minecraft.entity.player.PlayerInventory.insertStack(PlayerInventory.java:274)
    at net.minecraft.entity.player.PlayerInventory.insertStack(PlayerInventory.java:270)
    at team.comofas.arstheurgia.blocks.table.TableBlock.afterBreak(TableBlock.java:86)
```

I think in some cases you are trying to insert a `null` stack in the `PlayerInventory`. There, `placedItem` should probably be set to `ItemStack.EMPTY` before returning. Or you can set it to `ItemStack.EMPTY` in the constructor, same thing!
https://github.com/comofastm/ArsTheurgia/blob/master/src/main/java/team/comofas/arstheurgia/blocks/table/TableBlockEntity.java#L45

**Comment by dandin87** [comment]
More info, two tables, 1 sitting on top of another, they think they mined the bottom one

**Comment by dandin87** [comment]
https://hastebin.com/enoxocewom.yaml Here is a more complete and mapped crashlog.

**Comment by cominixo** [comment]
Thanks, I'll look into it.

**Comment by cominixo** [comment]
Fixed in 5ae4c99839908e9f757a8eea37895f37d272bf12 , will be in next release.
